recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -159,6 +159,9 @@ do_deploy() {
     if [ "${ENABLE_UART}" = "1" ]; then
         echo "# Enable UART" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
         echo "enable_uart=1" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+    else
+        echo "# Disable UART" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
+        echo "enable_uart=0" >>${DEPLOYDIR}/bcm2835-bootfiles/config.txt
```

Comment:
I'm wondering if setting an explicit disable would make more sense. This is because I think the current assumption is that when we don't set `ENABLE_UART` the value ends up being whatever the firmware uses by default. @kraj @pbrkr What do you guys think?

Reply:
it should be that enable/disable knob overrides firmware defaults regardless

Reply:
I'm not sure I understand what you mean. All I'm saying is that with this changing we move from default whatever the firmware states per machine -> default force disable.

Reply:
To propose an alternative, the value of "ENABLE_UART" can "1" to explicitly enable, "0" to explicitly disable, "" [empty] to keep default.

Reply:
That would work for me.

Reply:
Anyone else ?